#author : AnhTuan 27/5/2020
#use for node   : getPoseRobot_v4.cpp
#                 getPoseAruco_v2.cpp

link_file      : /home/amr1-l300/catkin_ws/src/sti_module/data/data_tag.txt

map_frame      : /map
tag_fistName   : /tag_
# cam_name       : /camera_aruco
cam_name       : /d435_color_optical_frame
robot_frame    : base_footprint

f_cam_name     : /f_camera_aruco  
f_tag_fistName : /f_tag_
f_robot_frame  : /f_robot

sl_filter      : 40